content/spin/v2/http-outbound.md

```diff
@@ -12,6 +12,7 @@ url = "https://github.com/fermyon/developer/blob/main/content/spin/v2/http-outbo
 - [Making HTTP Requests Within an Application](#making-http-requests-within-an-application)
   - [Local Service Chaining](#local-service-chaining)
   - [Intra-Application HTTP Requests by Route](#intra-application-http-requests-by-route)
+  - [Private Endpoints](#private-endpoints)
```

Comment:
We should probably be mentioning these in the HTTP trigger page because that's where app developers will actually set up private endpoints.  Calling them is kind of already covered by service chaining, so we probably just need a mention in the service chaining docs about "this is the only way to call private endpoints" or something.

Reply:
Thanks @itowlson - updating this now.